examples/README.md

```diff
@@ -204,7 +204,19 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>EfficientNet V2 B0</td>
     <td>Image Recognition</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/image_recognition/SavedModel/quantization/ptq">SavedModel</a></td>
+    <td><a href="./tensorflow/image_recognition/SavedModel/efficientnet_v2_b0/quantization/ptq">SavedModel</a></td>
+  </tr>
+  <tr>
+    <td>MobileNetV1_Saved</td>
```

Comment:
We don't add those models into readme before, because it's performance only and perf ratio is not good.  
![image](https://user-images.githubusercontent.com/51692656/212002598-5445ff80-5aa0-443c-8381-28ff1abbaa07.png)